pontoon/projects/management/commands/send_review_notifications.py

```diff
@@ -42,9 +42,9 @@ def get_description(self, author, notifyData):
                 if len(rejected) > 0:
                     msg += f", {len(rejected)} Rejected"
 
-            desc += f'\n* {project.name} ({locale.name} · {locale.code}): <a href="{url}">{msg}</a>'
+            desc += f'\n<li>{project.name} ({locale.name} · {locale.code}): <a href="{url}">{msg}</a></li>'
```

Comment:
After testing this layout a bit, I propose we make two changes:
1. Drop Locale name and only leave locale code in the brackets. That should make notification data easier to digest and more condensed. Also, some locale names are long. As in 4 words long.
2. Use `PROJECT (LOCALE)` piece as anchor instead of the `msg` piece.

Reply:
```suggestion
            desc += f'\n<li><a href="{url}">{project.name} ({locale.code})</a>: {msg}</li>'
```